cluster/src/test/java/org/apache/iotdb/cluster/query/BaseQueryTest.java

```diff
@@ -68,6 +69,8 @@ protected static void checkAggregations(List<AggregateResult> aggregationResults
 
   @Before
   public void setUp() throws Exception {
+    IoTDBDescriptor.getInstance().getConfig()
+        .setCompactionStrategy(CompactionStrategy.NO_COMPACTION);
```

Comment:
Why changing the compaction strategy, if we don't change, will this test failed?